code-gen/parse-csl/src/Csl/Gen/Utils.hs

```diff
@@ -0,0 +1,230 @@
+module Csl.Gen.Utils (
```

Comment:
To improve modularity, consider extracting all domain-specific functionality to different modules, and only use `Csl.Gen.Utils` for very general functions like `withSemicolon` or `wrapText`. I would suggest putting those new modules into the `Csl.Gen.Lib` namespace.

Reply:
Addressed in f2f7472.